### DISCUSSION TITLE
original SVG file for blocks.jpg

### DISCUSSION THREAD

**Discussion by Steve0Greatness** [FlagClicked/FlagClicked#87] [Help]
does anyone have it?

**Comment by Steve0Greatness** [comment]
@TheColaber it seems you were the one to add it, do you have the original file? Or is it just a jpg?

**Reply by danielgallab** [reply]
Sorry for not responding, actually, Ryker gave me the image on discord and I've asked him to create a new one but with transparency and and an svg
